notifier/notifier.go

```diff
@@ -211,27 +211,32 @@ func (notifier *StandardNotifier) runSender(sender moira.Sender, ch chan Notific
 			if metric, found := notifier.metrics.SendersOkMetrics.GetRegisteredMeter(pkg.Contact.Type); found {
 				metric.Mark(1)
 			}
-		} else {
-			switch e := err.(type) {
-			case moira.SenderBrokenContactError:
+			continue
+		}
+		switch e := err.(type) {
+		case moira.SenderBrokenContactError:
+			log.Warning().
+				Error(e).
+				Msg("Cannot send to broken contact")
+			if metric, found := notifier.metrics.SendersDroppedNotifications.GetRegisteredMeter(pkg.Contact.Type); found {
```

Comment:
Сожет отдельный метод сделаем? Чтобы было

```suggestion
			notifier.metrics.MarkSendersDroppedNotifications(pkg.Contact.Type)
```

Reply:
соорудила